src/db/controller/packetFee.ts

```
@@ -2,20 +2,24 @@ import { DB } from '..'
 import { Coin, Coins } from '@initia/initia.js'
+import { createLoggerWithPrefix } from 'src/lib/logger'
 
 export class PacketFeeController {
   static tableName = 'packet_fee'
+  private static logger = createLoggerWithPrefix('[PacketFeeController] ');
 
   public static feedEvents(
     chainId: string,
     events: PacketFeeEvent[]
   ): () => void {
+    this.logger.info(`feedEvents: chainId=${chainId}, events.length=${events.length}`);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)